tests/testsuite/lints_table.rs

```diff
@@ -150,6 +150,7 @@ fn invalid_type_in_lint_value() {
 }
 
 #[cargo_test]
+#[ignore = "currently broken by serde(flatten) on TomlLintConfig"]
```

Comment:
Please fix

Reply:
Done, but I'm not 100% satisfied with the result.